CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Contribution Guide
+
+The infrastructure project contains:
+
+1. The [Ansible](ansible) scripts for bootstrapping the build and test hosts (including a way to test Ansible)
```

Comment:
Minor nit - the Ansible link goes to the Ansible guide, but the text says Ansible scripts, might be best to point to the folder that contains the scripts